public/assets/flights-336b8ef937e2508ca6e656084c82c1007ee0ba42e7a21a5668697c5ce52afd9a.js

```diff
@@ -0,0 +1,6 @@
+
+
+$(document).ready(function(){
+      $('.slider').slider({full_width: true, height: "40em"});
```

Comment:
Identifier 'full_width' is not in camel case.<br>Mixed double and single quotes.